pkg/edgeview/src/tcp.go

```diff
@@ -389,6 +439,187 @@ func setAndStartProxyTCP(opt string) {
 	}
 }
 
+// Generate a random 32-byte symmetric key for kubeconfig file encryption
+func generateSymmetricKey() ([]byte, error) {
+	key := make([]byte, 32) // 256-bit key for AES-256
+	_, err := rand.Read(key)
+	if err != nil {
+		return nil, err
+	}
+	hexKey := hex.EncodeToString(key) // 64-character hexadecimal string
+
+	// Generate a random byte
+	var b [1]byte
+	_, err = rand.Read(b[:])
+	if err != nil {
+		return nil, err
+	}
+
+	// Convert the byte to an integer and take the modulus to get it within the range 0-31
+	start := int(b[0]) % 32
+
+	// Return a 32-character substring starting from the random index
+	return []byte(hexKey[start : start+32]), nil
```

Comment:
This returns a key that has 256bits but from the entropy point-of-view it only has 128bits.
@shjala is that a problem for AES?